dojson/overdo.py

```diff
@@ -55,7 +55,9 @@ def do(self, blob):
 
         for key, value in iteritems(blob):
             for name, creator, field in self._query(key):
-                output[name] = creator(output, key, value)
+                val = creator(output, key, value)
```

Comment:
I do not like this approach as KeyError is usually generated when we get the value. In this case is when we set the value.

An other approach can be similar as utils.filter_value. But I don't see how to implement this as a decorator. Any idea?